tests/suites/test_suite_psa_crypto_pake.data

```diff
@@ -1,203 +1,204 @@
 PSA PAKE: uninitialized access to psa_pake_operation_t
-depends_on:PSA_WANT_KEY_TYPE_ECC_KEY_PAIR:PSA_WANT_ECC_SECP_R1_256:PSA_WANT_ALG_SHA_256
+depends_on:PSA_WANT_KEY_TYPE_ECC_KEY_PAIR:PSA_WANT_ECC_SECP_R1_256:MBEDTLS_HAS_ALG_SHA_256_VIA_MD_OR_PSA
```

Comment:
I am not sure this is the right thing to do. @mpg wdyt?

Reply:
Ok, that one is related to the on-going driver-only hashes work: this shouldn't be the right thing to do, but right now I'm afraid it is. One more PR from the on-going work, and we can revert to `PSA_WANT_ALG_SHA_256` which is morally right (PSA things should only use `PSA_WANT` dependencies).

One of the motivations of the on-going work is precisely to avoids things like that where you need to use a feature macro that feels wong - and on top of that the only way to know which macro to use it to look at the code.

Reply:
ok thanks Manuel. @mprse is it the case that if we do not change that we have a CI failure? If this is not the case I would not bother to change it.